beacon/goclient/goclient.go

```diff
@@ -24,9 +26,15 @@ import (
 	beaconprotocol "github.com/bloxapp/ssv/protocol/v2/blockchain/beacon"
 )
 
-// DataVersionNil is just a placeholder for a nil data version.
-// Don't check for it, check for errors or nil data instead.
-const DataVersionNil spec.DataVersion = math.MaxUint64
+const (
+	// DataVersionNil is just a placeholder for a nil data version.
+	// Don't check for it, check for errors or nil data instead.
+	DataVersionNil spec.DataVersion = math.MaxUint64
+
+	// Client timeouts.
+	commonTimeout = time.Second * 5
+	maxTimeout    = time.Second * 120
```

Comment:
IMO "max" is not the best description of this timeout, I'd prefer something like `longTimeout` because when I saw "common" and "max" first time I didn't understand what's the purpose of "max" just from its name. But that's subjective, you might disagree here

Reply:
https://github.com/bloxapp/ssv/blob/a25c4fb942b7c1a2d8b9ab0c74821ab19b4e45a9/beacon/goclient/goclient.go#L34-L36